docs/quickstart.rst

```diff
@@ -115,7 +115,7 @@ Jira Webhook
 |              | | The ``<url_for_sg_jira_bridge>`` is the host name or IP address of the computer you   |
 |              | | will be launching ``webapp.py`` or ``service.py`` from.                               |
 +--------------+-----------------------------------------------------------------------------------------+
-| Description  | "Webhook that syncs Jira data with ShotGrid using the SG Jira Bridge"                   |
+| Description  | "Webhook that syncs Jira data with Flow Production Tracking using the SG Jira Bridge"                   |
```

Comment:
Probably it's better to have

```suggestion
| Description  | "Webhook that syncs Jira data with Flow Production Tracking using the Jira Bridge"                   |
```

Otherwise, I think writing Flow Production Tracking twice won't look good.